src/app/clusters/service-area-server/service-area-delegate.cpp

```diff
@@ -0,0 +1,101 @@
+#include "service-area-delegate.h"
+#include "service-area-server.h"
+
+using namespace chip::app::Clusters::ServiceArea;
+
+bool Delegate::GetSupportedLocationById(uint32_t aLocationId, uint32_t & listIndex, LocationStructureWrapper & aSupportedLocation)
+{
+    bool ret_value = false;
```

Comment:
please directly return as needed rather than this top level variable. This style of code makes following code and maintenance harder (e.g. when reading the return, I have to go through and see what the original assignment is and I need to follow through the entire code path for all changes).

Reply:
These have been removed [here](https://github.com/project-chip/connectedhomeip/pull/33991/commits/683b0fe4bf1ed901ea7d48a9c41fa5c59ccd921a).